### PR TITLE
Add Hetzner storage-box support

### PR DESCRIPTION
**Pull request by vaisov**
Faced issues when using this role on Hetzner Storage Box. This PR adds a way to manage keys on Hetzner Storage Boxes without breaking original workflow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,12 +39,13 @@ SSH key based authentication configured.
   hosts: localhost
 
   roles:
-    - role: ryankwilliams.ssh_copy_id
+    - role: rywillia.ssh-copy-id
```
Comment (ryankwilliams) [inline README.md]: Could this line be reverted please? I recently renamed my GitHub account and the role on galaxy that will include the latest changes is `ryankwilliams.ssh_copy_id`.

**Comment by ryankwilliams** [comment]
@vaisov Thank you for your PR!

File: README.md (hunk 1 above)
Comment (vaisov) [reply README.md]: Sure, reverted

**Comment by ryankwilliams** [comment]
LGTM

**Pull request merged**